vacasa/connect/connect.py

```diff
@@ -523,3 +523,36 @@ def get_reservations(self, params: dict = None):
         headers = self._headers()
 
         return self._iterate_pages(url, headers, params)
+
+    def get_quote_by_id(self, unit_id: int, arrival_date: str, departure_date: str, adult_count: int, params: dict = None):
+        """Retrieve a quote for a unit_id
+
+        Yields:
+            A dict containing attributes about the requested quote.
+        """
+
+        if params is None:
+            params = {}
+
+        if unit_id is None:
+            raise TypeError
+
+        if arrival_date is None:
+            raise TypeError
+
+        if departure_date is None:
+            raise TypeError
+
+        if adult_count is None:
+            raise TypeError
+
+        params['unit_id'] = unit_id
+        params['arrival'] = arrival_date
+        params['departure'] = departure_date
+        params['adults'] = adult_count
+
+        url = f"{self.endpoint}/v1/quotes"
+        headers = self._headers()
+
+        r = self._get(url, headers, params)
+        return r.json()['data']
```

Comment:
Any particular reason you're not using `_iterate_pages()` to yield a generator here? It's fine if there's a reason not to but it's different from the other SDK methods.

Reply:
Nevermind...I realize now you're just getting a single quote so there's no need to iterate over results here. In that case I would change 'Yields' in the docstring to 'Returns'.